virtme/commands/run.py

```diff
@@ -1063,7 +1063,7 @@ def do_it() -> int:
             qemuargs.extend(["-chardev", "file,path=/proc/self/fd/2,id=dmesg"])
             qemuargs.extend(["-device", arch.virtio_dev_type("serial")])
             qemuargs.extend(["-device", "virtconsole,chardev=dmesg"])
-            kernelargs.extend(["console=hvc0"])
+            kernelargs.extend(["console=ttyS0", "debug"])
```

Comment:
@arighi just to be sure, was the `console=` modification expected? I don't think that's an issue to switch to `ttyS0`, just surprised :)